runtime-modules/content/src/lib.rs

```diff
@@ -1371,22 +1371,22 @@ impl<T: Trait> Module<T> {
     /// Ensure Channel Migration Finished
 
     /// Ensure Video Migration Finished
-    fn ensure_migration_done() -> DispatchResult {
+    fn ensure_migration_done() -> bool {
```

Comment:
with the change of return type a better name for the function would now be `is_migration_done() -> bool`